sklearn/multioutput.py

```diff
@@ -905,6 +954,12 @@ class ClassifierChain(MetaEstimatorMixin, ClassifierMixin, _BaseChain):
 
         .. versionadded:: 1.2
 
+    base_estimator : estimator, default=None
```

Comment:
```suggestion
    base_estimator : estimator, default="deprecated"
```

Reply:
> Why do you propose default="deprecated" in the docstring

Since `None` is not use currently used as an option (`base_estimator` is a positional argument), we can use directly `"deprecated"` to detect whether or not a user is passing an estimator to the `base_estimator`.

The `None` + `"deprecated"` is usually used when the parameter take already an option that default to None.

Here, I pretty much follow the guide provided in the documentation: https://scikit-learn.org/dev/developers/contributing.html#deprecation (Sect. "Deprecating a parameter").

Reply:
> Since None is not use currently used as an option (base_estimator is a positional argument), we can use directly "deprecated" to detect whether or not a user is passing an estimator to the base_estimator.

Oh yes, that makes sense. Thanks for explaining.